static/docs/src.html

```diff
@@ -331,6 +331,10 @@ <h2>CSS classes</h2>
       <section data-include="practicelab"></section>
       <section data-include="remove"></section>
       <section data-include="removeOnSave"></section>
+      <section>
+        <h3>W3C Specific</h3>
+        <section data-include="updateable-rec"></section>
```

Comment:
Should add an id matching what we referenced in ReSpec.
```suggestion
        <section data-include="updateable-rec" id="updateable-rec-class"></section>
```